android/BUILD.gn

```diff
@@ -17,5 +18,6 @@ java_cpp_template("brave_config_java") {
   defines = [
     "BRAVE_ANDROID_DEVELOPER_OPTIONS_CODE=\"$brave_android_developer_options_code\"",
     "BRAVE_ANDROID_P3A_ENABLED=$brave_p3a_enabled",
+    "BRAVE_SAFEBROWSING_API_KEY=\"$brave_safebrowsing_api_key\"",
```

Comment:
Do we have a different API key for the SafetyNet attestation API? Or is it the same key as for SafetyNet Safe Browsing?

Reply:
we have 2 diff keys for that. We can use one key, I just created another one to avoid disruption of 2 systems at once(rewards and safe browsing) in case of compromising one of it.